osgp/shared/osgp-kafka-distributionautomation/src/test/java/org/opensmartgridplatform/adapter/kafka/da/signature/MessageSignerTest.java

```diff
@@ -34,6 +34,8 @@
 
     private static final boolean IS_SIGNING_ENABLED = true;
 
+    private static final boolean IS_STRIP_HEADERS = true;
```

Comment:
I thought that our convention was to use the `is` prefix only for getters, not for fields?